lca-main.yaml

```diff
@@ -18,6 +18,10 @@ Transform: AWS::Serverless-2016-10-31
 Description: Amazon Transcribe Live Call Analytics with Agent Assist - LCA (v0.7.1) (SO9072)
 
 Parameters:
+  ExpirationInDaysForIASTACK:
```

Comment:
It's AIStack, not IAStack. But regardless, let's keep the parameter name consistent.. eg `RecordingsBucketRetentionDays` as I suggested above.